### DISCUSSION TITLE
Tag Detection Specifics

### DISCUSSION THREAD

**Discussion by EceChaik** [larics/icuas22_competition#50] [Q&A]
We have a question regarding the target marker. Currently our implementation assumes that it will be the same marker (id 4) with the same dimensions (20cm x 20cm), which helps with the detection. 

Can we expect the evaluation environment to have the same marker? Or does the competition require solutions that can detect markers of any library / dictionary and of any dimensions?

**Comment by fpetric** [comment]
Hi @EceChaik, 

it will be the same marker, at least for the simulation phase.
